bedrock/firefox/templates/firefox/australis/growth-firstrun-test2.html

```diff
@@ -0,0 +1,163 @@
+      <div id="ui-initial">
+        <div class="logo"></div>
+        <div class="stage">
+          <h1>{{ _('Welcome to Firefox') }}</h1>
+          <p>{{ _('Committed to you, your privacy and an open Web.') }}</p>
+        </div>
+
+        <div class="mask-content mask-hidden">
+          <header>
+            <h2 id="heading1">{{ _('Try these add-ons in one click') }}</h2>
+            <h2 id="heading2">
+              {{ _('Manage your add-ons <a id="highlight-addons" href="#" role="button">here</a>.') }}
+            </h2>
+          </header>
+
+          <ul id="mask-addons">
+            <li id="ghostery">
+              <i class="icon icon-ghostery"></i>
+
+              <div class="addon-content">
+                <strong>{{ _('Ghostery') }}</strong>
+
+                <p>
+                  {{ _('Protect your privacy.') }}
+                  {{ _('See who’s tracking your web browsing and block them with Ghostery.') }}
+                </p>
+
+                <div class="iframe-wrapper">
+                  <div class="iframe-overlay">{{ _('Added to toolbar') }}</div>
+                  <iframe class="addon" data-src="//addons-dev.allizom.org/static/html/mozorg/ghostery.html" frameborder="0" scrolling="no"></iframe>
+                </div>
+
+                <div id="yt-ghostery" class="youtube"></div>
+              </div>
+            </li>
+            <li id="lightbeam">
+              <i class="icon icon-lightbeam"></i>
+
+              <div class="addon-content">
+                <strong>{{ _('Lightbeam') }}</strong>
+
+                <p>
+                  {{ _('See interactive visualizations that show you the first and third party sites you interact with on the web.') }}
+                </p>
+
+                <div class="iframe-wrapper">
+                  <div class="iframe-overlay">{{ _('Added to toolbar') }}</div>
+                  <iframe class="addon" data-src="//addons-dev.allizom.org/static/html/mozorg/lightbeam.html" frameborder="0" scrolling="no"></iframe>
+                </div>
+
+                <div id="yt-lightbeam" class="youtube"></div>
+              </div>
+            </li>
+          </ul>
+
+          <div id="mask-links">
+            <a class="what-is-addon" rel="external" href="https://support.mozilla.org/kb/find-and-install-add-ons-add-features-to-firefox">{{ _('What is an add-on?') }}</a>
+          </div>
+        </div>{#-- /.mask-content --#}
+      </div>{#-- /#ui-initial --#}
+
+      <div id="ui-final">
+        <h3>{{ _('Looking for more add-ons?') }}</h3>
+
+        <p>
+          {{ _('You can find them in this menu when you need them.') }}
+        </p>
+
+        <ul>
+          <li>
+            <a class="what-is-addon" rel="external" href="https://support.mozilla.org/kb/find-and-install-add-ons-add-features-to-firefox">{{ _('What is an add-on?') }}</a>
+          </li>
+          <li>
+            <a role="button" href="#" id="thanks-exit">{{ _('Thanks!') }}</a>
```

Comment:
I know this is only a test, but this could probably just be a `<button>`. Up to you to change it :smile: